e2e/dractions/crud.go

```diff
@@ -188,9 +188,15 @@ func createPlacementManagedByRamen(ctx types.TestContext, name, namespace string
 	return nil
 }
 
-func generateDRPCDiscoveredApps(name, namespace, clusterName, drPolicyName, placementName,
-	appname, protectedNamespace string,
+func generateDRPCDiscoveredApps(clusterName, drPolicyName,
+	appname string, ctx types.TestContext,
+	recipeRef *ramen.RecipeRef,
 ) *ramen.DRPlacementControl {
```

Comment:
If we cannot fit the function in one line it should be formatted in one argument per line.

But this is also as sign that we have too many argument and we need an Options type to pass them in a more clear way.